### PR TITLE
[Localization] Change Localization.country

### LINKED ISSUE #4540: [android] Localization.country return country part of locale instead of region
### Environment
```
  Expo CLI 2.18.4 environment info:
    System:
      OS: macOS 10.14.5
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.7.0 - ~/.nvm/versions/node/v10.7.0/bin/node
      Yarn: 1.13.0 - /usr/local/bin/yarn
      npm: 6.1.0 - ~/.nvm/versions/node/v10.7.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      @types/expo: ^32.0.0 => 32.0.0 
      @types/react: ^16.8.1 => 16.8.1 
      @types/react-native: ^0.57.32 => 0.57.32 
      expo: ^32.0.0 => 32.0.4 
      react: 16.5.0 => 16.5.0 
      react-native: https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz => 0.57.1 
    npmGlobalPackages:
      expo-cli: 2.18.4
```

Android 6.

### Steps to Reproduce

I use `Localization.country` from `expo-localization` to get the user's country. 

I expect this field to return the configured country. For example, when configuring Region as in the following screenshot from the Android preferences, "UK" should be returned indicating Ukraine: 
preferences region.

Instead, I get part of the configured locale. For example, when the locale is set to English (US), as on the following screenshot, I get US as country. 
preferences locale

On iOS it works as expected, returning country.

### PR DESCRIPTION
**Pull request by lukmccall**
# Why

Resolve #4540.

# How

Country variable on Android came from currently selected language, however on devices with iOS country was got from `Region` in `Language & Region`. 
Not all android phone provide `Region` setting, so I suggest renaming Localization.country to Localization.region and make it available only on iOS.

# Test Plan

You can check it on https://snack.expo.io/@lukaszkosmaty/9e807b.
I also updated test-suits.

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/localization.md [hunk 1]
```diff
@@ -57,9 +57,9 @@ Native device language, returned in standard format. Ex: `en`, `en-US`, `es-US`.
 
 List of all the native languages provided by the user settings. These are returned in the order the user defines in their native settings.
 
-#### `Localization.country: ?string`
+#### `Localization.region: ?string`
 
-Country code for your device.
+**Available on iOS only.**  Region code for your device which came from `Region` setting in `Language & Region`. Ex: `US`, `NZ`.
```
Comment (EvanBacon) [inline docs/pages/versions/unversioned/sdk/localization.md]: Seems like web has support as well

**Review by bbarthec — COMMENTED** [review]
LGTM 💯 Please add entry in `CHANGELOG.md` under `breaking changes` section 😃

**Review by sjchmiela — APPROVED** [review]

**Review by bbarthec — APPROVED** [review]

**Review by tsapeta — CHANGES_REQUESTED** [review]
Looks good, but could you add changelog entry and fix issues found by CI in `expo_sdk` job?

**Review by tsapeta — APPROVED** [review]

**Comment by tobiascornille** [comment]
So it's not possible to guess the country on Android without asking for the user's location?

**Comment by Stevemoretz** [comment]
Weird though how does Whatsapp know your phone-number prefix then?

**Pull request merged**